flutter_todo/lib/screens/log_in.dart

```diff
@@ -9,18 +8,19 @@ class LogIn extends StatefulWidget {
 }
 
 class _LogInState extends State<LogIn> {
-  bool _logIn = true;
+  bool _isLogin = true;
```

Comment:
note: i've noticed that there's some variation btwn private variables (prefixed w `_`) and public variables. this more my fault than yours in this project.

can/should we standardize how we define variables?

Reply:
I will check them all again, but the rules that I followed are:
If the variable is inside a method (for example inside `builder` method) we don't have to make it private with adding prefix "_". They are available only inside the method and cannot be accessed outside the method.
If it is a class member that is going to be accessed only in the class it should be private "_".
If it is a class member that is used in other classes it should be public - without "_". Like the members of the  `RealmServices` for example.